test/shared/src/main/scala/zio/test/SuiteConstructor.scala

```diff
@@ -7,76 +7,72 @@ import zio.stm.ZSTM
 trait SuiteConstructor[In] {
   type OutEnvironment
   type OutError
-  type OutSuccess
-  def apply(spec: In)(implicit trace: ZTraceElement): Spec[OutEnvironment, OutError, OutSuccess]
+  def apply(spec: In)(implicit trace: ZTraceElement): Spec[OutEnvironment, OutError]
 }
 
 object SuiteConstructor extends SuiteConstructorLowPriority1 {
 
-  type WithOut[In, OutEnvironment0, OutError0, OutSuccess0] =
+  type WithOut[In, OutEnvironment0, OutError0] =
     SuiteConstructor[In] {
       type OutEnvironment = OutEnvironment0
       type OutError       = OutError0
-      type OutSuccess     = OutSuccess0
     }
 
-  implicit val NothingConstructor: SuiteConstructor.WithOut[Nothing, Any, Nothing, Nothing] =
+  implicit val NothingConstructor: SuiteConstructor.WithOut[Nothing, Any, Nothing] =
     new SuiteConstructor[Nothing] {
       type OutEnvironment = Any
       type OutError       = Nothing
       type OutSuccess     = Nothing
-      def apply(spec: Nothing)(implicit trace: ZTraceElement): Spec[Any, Nothing, Nothing] =
+      def apply(spec: Nothing)(implicit trace: ZTraceElement): Spec[Any, Nothing] =
         Spec.multiple(Chunk.empty)
     }
 }
 
 trait SuiteConstructorLowPriority1 extends SuiteConstructorLowPriority2 {
 
-  implicit def SpecConstructor[R, E, T]: SuiteConstructor.WithOut[Spec[R, E, T], R, E, T] =
-    new SuiteConstructor[Spec[R, E, T]] {
+  implicit def SpecConstructor[R, E, T]: SuiteConstructor.WithOut[Spec[R, E], R, E] =
```

Comment:
Looks like we can delete the `T` type param here too

Reply:
Good catch! Deleted.